CRM/MembershipExtras/Hook/Alter/CalculatedMembershipStatus.php

```diff
@@ -69,52 +77,33 @@ public function alterMembershipStatus(&$calculatedStatus, $arguments, $membershi
     $this->membership = $membership;
     $this->calculationArguments = $arguments;
 
+    // If membership is being created, it should start in 'Pending' status.
```

Comment:
hmm, I only meant that membership added through "manage installments" should be defaulted to "pending" since it is a special case, but we don't want every membership created to start pending  (what if I create a membership with one completed payment for example ?) .

I think the best and most simple approach here is just to update the api call that create the membership in RM/MembershipExtras/Form/RecurringContribution/AddMembershipLineItem.php  class so it always pass "Pending" as the membership status.